src/main/java/com/saralein/server/response/FileHelper.java

```diff
@@ -4,7 +4,8 @@
 import java.util.List;
 
 public interface FileHelper {
```

Comment:
Be wary of `*Helper`. They can end up being a dumping ground for unrelated things. Be vigilant in what you put here, and be on the lookout for a more specific name.